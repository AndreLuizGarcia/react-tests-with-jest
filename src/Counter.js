import React, { Component } from 'react'

class Counter extends Component {
  render(){
    return(
      <div className="Counter">
        <h2>VocÃª tem { this.props.count } counters;</h2>
      </div>
    )
  }
}

export default Counter