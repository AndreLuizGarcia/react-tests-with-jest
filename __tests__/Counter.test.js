import React from  'react'
import Counter from '../src/Counter'
import renderer from 'react-test-renderer'

describe('Testing Counter Component', () => {
  it('render correctly', () => {
    const wrapper = renderer
      .create(<Counter count={5} />)
      .toJSON()

    expect(wrapper).toMatchSnapshot()
  })
})