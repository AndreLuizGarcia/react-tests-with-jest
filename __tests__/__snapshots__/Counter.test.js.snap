// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Counter Component render correctly 1`] = `
<div
  className="Counter"
>
  <h2>
    VocÃª tem 
    5
     counters;
  </h2>
</div>
`;
